internal/iso/iso_test.go

```diff
@@ -18,8 +23,20 @@ func TestReqPathInvalid(t *testing.T) {
 	for name, tt := range tests {
 		u, _ := url.Parse(tt.isoURL)
 		t.Run(name, func(t *testing.T) {
+			// Will print 10% of entries.
```

Comment:
this should be 2% ? also might not be needed in a test that only checks for error responses.

Reply:
yeah, fair. this was just copy paste so that the `SampleLogger` wasn't nil and panicking the service. I'll update this.